pysys/basetest.py

```diff
@@ -69,11 +69,6 @@ class BaseTest(ProcessUser):
 	individual validation outcomes. Several potential outcomes are supported by the PySys framework 
 	(C{SKIPPED}, C{BLOCKED}, C{DUMPEDCORE}, C{TIMEDOUT}, C{FAILED}, C{NOTVERIFIED}, and C{PASSED}) and the 
 	overall outcome of the testcase is determined using aprecedence order of the individual outcomes. 
-	
-	All C{assert*} methods except for C{assertThat} support variable argument lists for common non-default parameters.
```

Comment:
Might still be worth mentioning that all of the `assert*` methods except for `assertThat` have `abortOnError` and `assertMessage` parameters and what these do?

Reply:
Hmmm not sure that would make things clearer - the best place to docuemnt what each method does is on the methods themselves, whcih is where people will look for that detail, not at the containing class. The need to write things like "all of the assert* methods except for assertThat have ..." is an added reason against adding per-method info at the class level (e.g. keeping that statement up to date when we add new assert* methods could be painful). Is that ok?

Reply:
I guess I still feel there is some behaviour shared by a lot of methods that would be worth mentioning at the class level, even if you don't want to explicitly list the methods it applies to (agree with you on that part for sure).  Although the more I think about it the more I'm not sure it's really that important :-)  I'll leave it up to you - no objections if you want to leave it as it is.

Reply:
Thanks. It's clearly a subjective one; since your opinion wasn't so strong I'm going to keep it like this as I feel this helps the docuemntation be clear and easy to keep maintained.